README.md

```diff
@@ -736,6 +736,25 @@ use the bleeding edge version of the logic. Then simply set the following config
 SitePrism.use_all_there_gem = true
 ```
 
+### Getting the list of missing element
+If all_there? returns false and you wish to get the list of missing elements for debugging purposes
+you may want to use `elements_missing` method. It will return the names of missing elements out of
+expected_elements list
+```ruby
+class TestPage < SitePrism::Page
```

Comment:
Can we use one of the ones above, or something that doesn't contain the word "Page", as it's something I think some people like but I personally don't. I've tried to remove traces of `SomethingSection` and `SomethingPage` from the readme